docs/3rd_party/cryptography/README.md

```diff
@@ -0,0 +1,24 @@
+# Cryptography
+
+The `cryptography` library is a robust toolkit for implementing secure encryption, decryption, and cryptographic protocols in applications.
+
+## Usage
+
+```python
+from cryptography.hazmat.primitives.asymmetric import rsa
```

Comment:
@OvidiuCode & @cmin764 good to check in this example what is the latests recommended encryption algorithm, key sizes etc. so that we don't guide anything obsolete.

Reply:
According to some SSH best practices (not sure if relevant), an RSA of 2048 or even 4096 bit is still considered secure.

Although there are these more exotic eliptic curves ones, like ECDSA which are stronger at brute-force. ([context](https://security.stackexchange.com/a/144044))
> Generating Ed25519 keys is faster compared to traditional RSA keys.

[Generating Ed25519 keys is faster compared to traditional RSA keys.](https://www.brandonchecketts.com/archives/its-2023-you-should-be-using-an-ed25519-ssh-key-and-other-current-best-practices)

So this seems to be the new kid on the block.